cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/FlowExtensions.kt

```diff
@@ -92,4 +96,28 @@ fun FlowLogic<Any>.getSchemaById(schemaId: SchemaId): StateAndRef<IndySchema>? {
     val result = serviceHub.vaultService.queryBy<IndySchema>(criteria)
 
     return result.states.firstOrNull()
-}
\ No newline at end of file
+}
+
+fun FlowLogic<Any>.getRevocationRegistryDefinitionById(id: RevocationRegistryDefinitionId): StateAndRef<IndyRevocationRegistryDefinition>? {
```

Comment:
May be all such state related extensions should be with schema?

Reply:
Ok